### ISSUE TITLE
QR Download is Broken

### ISSUE THREAD

**Issue by SylarBRG** [Steveice10/FBI#468]
I tried to download using QR codes and it does not allow it, it automatically leaves error and the .cia file that I try to install if it is available

**Comment by Sanaki** [comment]
If you're attempting to download from github using QR, I don't believe TLS1.2 support has been added to FBI yet. In that case, try using [QRaken](https://github.com/BernardoGiordano/QRaken/) for the time being. It's not perfect, but it'll do the job until we have that update.

**Comment by SylarBRG** [comment]
![img_20180825_193353](https://user-images.githubusercontent.com/41936578/44623810-b92f8b00-a89e-11e8-8617-f7fddd65b6c9.jpg)
![img_20180825_193400](https://user-images.githubusercontent.com/41936578/44623811-b9c82180-a89e-11e8-959d-cff6ccb34f07.jpg)
It used to work with Dropbox links

**Comment by Steveice10** [comment]
You aren't connected to the Internet. Note that the wireless status icon in the top right has an X over it.

**Comment by SylarBRG** [comment]
Sorry, I uploaded the wrong photo, but this is the correct one, the error persists.
I used to work with Dropbox links

![img_20180825_194518](https://user-images.githubusercontent.com/41936578/44623845-ed577b80-a89f-11e8-8f59-e960feadeb3e.jpg)

**Comment by Steveice10** [comment]
Alright, I'll look into it later today when I get the chance.

**Comment by SylarBRG** [comment]
![img_20180825_195252](https://user-images.githubusercontent.com/41936578/44623872-ca799700-a8a0-11e8-9bba-26fb7859bafb.jpg)
The same thing happens with other URLs

**Comment by SylarBRG** [comment]
Thank you for your attention, I will wait for the solution

**Comment by jvhellraiser** [comment]
it wont work on mediafire links either and i know for a fact they use to work.

**Comment by Steveice10** [comment]
Can you check if this build resolves the issue? I've tested it with Dropbox myself, and it seems to work now.

https://www.dropbox.com/s/aph4cjst9jtx63l/FBI.zip?dl=1

**Comment by SylarBRG** [comment]
Perfect, now if it works well, thank you
![img_20180827_143851](https://user-images.githubusercontent.com/41936578/44681751-1c3c3180-aa07-11e8-840e-bb4258c0b779.jpg)

**Comment by urherenow** [comment]
Seriously? You’re posting proof of piracy in public? You shouldn’t expect to be helped with this. Install from QR code is there for HOMEBREW.

**Comment by jvhellraiser** [comment]
oh yeah a copy from one of my so many back ups is that i want it to try the QR codes to see if it really works i uploaded mines to mediafire but they are not working , the person before me must took this from one of his back ups too. see.....
![mario-luigi-bowsers-inside-story-nintendo-ds](https://user-images.githubusercontent.com/16488761/44696590-2cbfcc80-aa46-11e8-8c5c-56951e3080f2.jpg)

**Comment by Steveice10** [comment]
Have you tried opening the link in a browser to make sure it actually leads directly to a CIA download, and not some web page?

**Comment by jvhellraiser** [comment]
Yes check: http://download2268.mediafire.com/znyb0mfdr4cg/53k8q6y26pyr1rd/Super+Metroid+old.cia?download=1 this is the direct link of that CIA but for some unknown reason FBI can read it and try put it in the address bar and click enter you will see the CIA downloading in direct link.

**Comment by Steveice10** [comment]
That link takes me to a page about regenerating a download key, not a direct download. And for the record, you really probably shouldn't be posting links to game CIAs here.

**Comment by jvhellraiser** [comment]
you what i am sorry from taking from your time the true is that i am really tired of trying for FBI to read Qr links, dropbox link work so whatever if you in the near future manage to add other websites to work that'll be cool. bye for now.

**Comment by jvhellraiser** [comment]
One last request any websites other than dropbox that let you direct link and work with FBI?

**Comment by Steveice10** [comment]
Just about any site that allows you to make a permanent direct download link should work, other than GitHub.
